resources/js/pages/Home.vue

```diff
@@ -2,98 +2,104 @@
     <div>
         <div class="home-grid">
             <div class="task-lists">
-                <div class="task-list">
-                    <h3>Task list 1</h3>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
-                </div>
-                <div class="task-list">
-                    <h3>Task list 2</h3>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
-                </div>
-                <div class="task-list">
-                    <h3>New Task list </h3>
+                <template v-for="(list, index) in taskLists">
+                    <task-list :key="index" :taskList="list" class="task-list border"></task-list>
+                </template>
+                <div class="task-list border">
+                    <h3>Create new task list</h3>
                 </div>
             </div>
 
-            <div class="character-tab">
-                <div class="character">
-                    <h3>Character</h3>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
+            <div class="right-align">
+                <div class="character-tab border">
+                    <div class="character">
+                        <h3>Character</h3>
+                        <p>Dummy text</p>
+                        <p>Dummy text</p>
+                        <p>Dummy text</p>
+                    </div>
                 </div>
-            </div>
 
-            <div class="friends-list">
-                <div class="friends">
-                    <h3>Friends</h3>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
-                    <p>Dummy text</p>
+                <div class="friends-list border">
+                    <div class="friends">
+                        <h3>Friends</h3>
+                        <p>Dummy text</p>
+                        <p>Dummy text</p>
+                        <p>Dummy text</p>
+                    </div>
                 </div>
             </div>
+            
         </div>
         
     </div>
 </template>
 
 
 <script>
+import {mapGetters} from 'vuex';
+import TaskList from '../components/TaskList.vue';
 export default {
+  components: { TaskList },
+    mounted() {
+        this.getTasks();
+    },
```

Comment:
de dispatch zou hier ook gelijk in kunnen, je gebruikt hem nergens anders meer in je code